src/features/builtin/ft_cd_utilities/ft_cd_change_dir.c

```diff
@@ -59,13 +55,17 @@ static void	manipulate_var(t_cd *cd)
 		ft_cd_set_env("PWD", "/");
 }
 
+/*
+**	step10 in posix doc
+*/
+
 int			ft_cd_change_dir(t_cd *cd)
 {
 	if (chdir(cd->curpath) == -1)
 	{
 		ft_dprintf(STDERR_FILENO, "%s: cd: No such file or directory\n", \
 		SHELL_NAME);
-		free(cd->curpath);
+		ft_strdel(&cd->curpath);
```

Comment:
Be careful with the ft_strdel or free some memory during processing the function, it will be better to reset it to Null to avoid double free when freeing the memory

Reply:
@xopxop  Thanks for pointing it out! I understand the double free issue that might be caused by reckless freeing. But just FYI, `ft_strdel()` does both freeing and setting the pointer to `NULL`. That's why it takes `char **`